src/notifications/conf/base.py

```diff
@@ -342,3 +342,7 @@
 # RabbitMQ
 BROKER_URL = os.getenv('PUBLISH_BROKER_URL', 'amqp://guest:guest@localhost:5672/%2F')
 CHANNEL = QueueChannel(params=BROKER_URL)
+
+# Celery
+CELERY_BROKER_URL = "amqp://127.0.0.1:5672//"
```

Comment:
these need to be pulled from the environment, with a fallback to defaults. See above ;)

Reply:
fixed. Used `os.getenv`